utils/inputValidation.ts

```diff
@@ -0,0 +1,25 @@
+type ValidationType = "email" | "password" | "hourlyPay" | "checkingPassword";
```

Comment:
```suggestion
type ValidationTarget = "email" | "password" | "hourlyPay" | "checkingPassword";
```
검증 대상이 어떤 타입이냐는 거니까 ValidationTarget 어떠신가요? 기존 이름이 모호한 것 같아서요